src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// DragAndDrop controls the drag objet and data.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class DragAndDrop : BaseHandle
+    {
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public DragAndDrop() : this(Interop.DragAndDrop.New(), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        internal DragAndDrop(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+
+        }
```

Comment:
이걸 이렇게 바꾸면 singletone 객체가 됩니다.
C#에 익숙하지 않으신것 같아서 어떻게 하면 되는지 작성해드렸습니다.
```suggestion
        private DragAndDrop() : this(Interop.DragAndDrop.New(), true)
        {
            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
        }

        private DragAndDrop(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
        {
        }
        
        public static DragAndDrop Instance {get;} = new DragAndDrop()
        
```

Reply:
수정하였습니다.